discovery-server.repos

```diff
@@ -1,13 +1,13 @@
 repositories:
-    eProsima/Fast-CDR:
+    fastcdr:
         type: git
         url: https://github.com/eProsima/Fast-CDR.git
         version: master
-    eProsima/Fast-DDS:
+    fastdds:
         type: git
         url: https://github.com/eProsima/Fast-DDS.git
         version: master
-    eProsima/Discovery-Server:
+    discovery-server:
```

Comment:
I think the criteria is to use underscore instead of hyphen. Hyphens conflicts with ROS 2 package generators (or something. I m not completely sure).